### ISSUE TITLE
verification issues on transition

### ISSUE THREAD

**Issue by 532910** [vector-im/riotX-android#1555]
Just updated to beta-0-91-1 from  GPlay.

1. On the startup it offers to verify this login, but the only method it to enter Recovery Passphrase or Key:
![image](https://user-images.githubusercontent.com/14617699/85913905-92975e00-b841-11ea-8684-d897e0ef7d05.png)

Other devices are not requested for the verification, possibly due to they already treat this devices as verified.

2. Trying to verify this device it ask for the method (manually, emoji...) selecting any of them does nothing, but later it starts show this  devices as verified.

3. People that are shown with the green shield on the desktop client has a red one on this updated device. Keys are shown as not trusted.

**Comment by 532910** [comment]
Only with the second attempt to clear storage I was able to verify it with  the desktop client and now it shows all verified contacs correctly.

**Comment by pengutronix-mirror** [comment]
I have the same problem. As no other device was allowing to start the verification (they all show it with a green shield already), I entered my recovery key. Now the updated devices shows all other contacts with a red shield.

**Comment by jluebbe** [comment]
I have the same problem. As no other device was allowing to start the verification (they all show it with a green shield already), I entered my recovery key. Now the updated devices shows all other contacts with a red shield. I've not cleared the storage yet.

**Comment by horstvogel** [comment]
I have the same problem, on the version v0.22.0 it is working, on riot-android-beta-0-91-1 it is **not** working

**Comment by Madic-** [comment]
I've the same problem and am only able to verify the device via recovery passphrase. But even then the messages aren't decrypted. All sessions are shown as trusted.
Only after the third time removing all data from the android app the image verification process was shown.
But still no luck. Riot always wants the recovery key, even if the session is verified. When trying to recover the keys, an error prompts that the Keys can't be recovered

**Comment by BillCarsonFr** [comment]
> I've the same problem and am only able to verify the device via recovery passphrase. But even then the messages aren't decrypted. All sessions are shown as trusted.
> Only after the third time removing all data from the android app the image verification process was shown.
> But still no luck. Riot always wants the recovery key, even if the session is verified. When trying to recover the keys, an error prompts that the Keys can't be recovered

thx, this is a separate issue, it has been fixed
